### DISCUSSION TITLE
Citations

### DISCUSSION THREAD

**Discussion by aiva005** [dendronhq/dendron#1217] [Q&A]
Hi!

I'm starting to use dendron and I need to use BiBLaTeX citations for academic work.

There's a Pandoc Citer extension that recognises my library path and can include the keys.

Enhanced Editor is **supposed** to be able to do citations via *pandoc* but there's a glitch that can be fixed by a [workaround here](https://github.com/shd101wyy/vscode-markdown-preview-enhanced/issues/432) thanks to @notZaki for pointing it out!

I follow all the instructions in the cited thread, and get to this point (https://github.com/shd101wyy/vscode-markdown-preview-enhanced/issues/432#issuecomment-906157058): 

I get the 

> [WARNING] Citeproc: citation lucaWantMakeBetter not found

which is weird, because I have this entry

`@online{lucaWantMakeBetter,`
`  title = {Want to {{Make Better Decisions}}? {{Start Experimenting}}},`
`  shorttitle = {Want to {{Make Better Decisions}}?},`
`  author = {Luca, Michael and Bazerman, Max H.},`
`  url = {https://sloanreview.mit.edu/article/want-to-make-better-decisions-start-experimenting/},`
`  urldate = {2021-08-23},`
`  abstract = {It’s early in the age of experimentation — and the right time to start building expertise.},`
`  langid = {american},`
`  organization = {{MIT Sloan Management Review}},`
`  keywords = {Decisions,Experiments,MIT},`
`  file = {C\:\\Users\\xyz\\Zotero\\storage\\JHM65QA7\\want-to-make-better-decisions-start-experimenting.html}`
`}`

and it's being picked up by another extension (pandoc citer) in the same .md file without any issues.

This is my header—which does NOT seem to be picked up by pandoc as a header **when I use pandoc as a parser in the Dendron Viewer Enhanced". (When I have the Enhanced Viewer to use another parser, it works fine—but I can't get citations, either.)

`---`
`id: blahblah`
`title: Test`
`desc: 'just testing stuff: primarily citations'`
`updated: 1629960547898`
`created: 1629775917010`
`bibliography: [C:\xyz\Dendron\vault\assets\bibliographies\global-library.bib]`
`---`

Basically, my understanding is that:
1. It does NOT recognise the .bib file, because...
2. It does NOT recognise the header syntax...?

I am using the Dendron versions of these extensions, but my understanding is that they work the same.

I am stumped as to what I'm doing wrong...

Does anyone have suggestions? Citations are a key feature for any student or professor, this seems like a key issue—I get a feeling (from the problems floating around) that I'm not the only one...?
